modules/ROOT/pages/design-ghs-install-authorize.adoc

```diff
@@ -0,0 +1,73 @@
+= Installing and Authorizing GitHub Synchronization
+
+Installing the GitHub Synchronization application in an organization is the first step in making the organization’s repositories accessible to API Designer.
+
+Install the GitHub Synchronization application within the GitHub user interface. Perform this action either as a GitHub organization administrator or as a GitHub user. Then, authorize the application when performing a GitHub Synchronization action for the first time within API Designer. Synchronization actions include creating new GitHub repositories for existing API specifications and creating new API specifications that are synchronized with existing GitHub repositories.
+
+== Install the GitHub Synchronization Application as a GitHub Organization Administrator
+
+To get started, first install the GitHub Synchronization application within the GitHub user interface.
+
+=== Before You Begin
+
+To install the GitHub Synchronization application as a GitHub organization administrator, you must have:
+
+* A GitHub account
+* A membership in the GitHub organization where the application will be installed
```

Comment:
Suggestion to avoid using 'will' 
A membership in the GitHub organization where you want to install the application